test/unit/tap-reporter.js

```diff
@@ -94,7 +93,7 @@ describe('Tap reporter', function () {
 
     emitter.emit('testEnd', data.actualFalsyTest)
 
-    expect(spy).to.have.been.calledWith('  actual: 0')
+    expect(spy).to.have.been.calledWith('  actual: "0"')
```

Comment:
No, `actual` is a YAML string that contains a JSON representation of the actual value. So in this case, it is supposed to be a YAML string that contains a JSON integer with value `0`.

Note that since this line is part of the test suite, to change this would require changing [the library code](https://github.com/jeberger/js-reporters/blob/master/lib/reporters/TapReporter.js#L49) so that it would not wrap integer values (and floating-point, boolean and `null`), and the same for the `expected` field. This would make the library code a lot more complex (and bug-prone) for very little gain.

Reply:
Okay, if this is what the spec recommends then it seems reasonable to me.